plugins/cpp_metrics/service/cxxmetrics.thrift

```diff
@@ -42,7 +88,33 @@ service CppMetricsService
     1:common.AstNodeId astNodeId)
 
   /**
-   * This function returns the names of metrics.
+   * This function returns all available C++ metrics
+   * for a particular module.
+   */
+  list<CppMetricsModule> getCppMetricsForModule(
+    1:common.FileId fileId)
+
+  /**
+   * This function returns all available C++ metrics
+   * (AST node-level) for a particular path.
+   */
+  list<CppAllMetricsAstNode> getCppAstNodeMetricsForPath(
+    1:string path)
+
+  /**
+   * This function returns all available C++ metrics
+   * (file-level) for a particular path.
+   */
+  list<CppAllMetricsModule> getCppFileMetricsForPath(
+    1:string path)
+
+  /**
+   * This function returns the names of AST node metrics.
+   */
+  list<CppAstNodeMetricsTypeName> getCppAstNodeMetricsTypeNames()
+
+  /**
+   * This function returns the names of module-level metrics.
    */
-  list<CppMetricsTypeName> getCppMetricsTypeNames()
+   list<CppModuleMetricsTypeName> getCppModuleMetricsTypeNames()
```

Comment:
Indentation error. (Extra space at the beginning of the line.)

Reply:
Fixed in e09e8ff80ab891cd8da090bdd74b5fe870ab6c1c.